device/tt_silicon_driver.cpp

```diff
@@ -642,199 +233,93 @@ std::vector<chip_id_t> ttkmd_scan() {
     return found_devices;
 }
 
-int get_config_space_fd(TTDevice *dev) {
-    if (dev->sysfs_config_fd == -1) {
-        static const char pattern[] = "/sys/bus/pci/devices/0000:%02x:%02x.%u/config";
-        char buf[sizeof(pattern)];
-        std::snprintf(buf, sizeof(buf), pattern,
-                      (unsigned int)dev->pci_bus, (unsigned int)dev->pci_device, (unsigned int)dev->pci_function);
-        dev->sysfs_config_fd = open(buf, O_RDWR);
+// bool is_hardware_hung(const PCIDevice *dev) {
+//     volatile const void *addr = reinterpret_cast<const char *>(dev->bar0_uc) + (dev->get_architecture_implementation()->get_arc_reset_scratch_offset() + 6 * 4) - dev->bar0_uc_offset;
+//     std::uint32_t scratch_data = *reinterpret_cast<const volatile std::uint32_t*>(addr);
 
-        if (dev->sysfs_config_fd == -1) {
-            dev->sysfs_config_fd = open(buf, O_RDONLY);
-        }
-    }
+//     return (scratch_data == 0xffffffffu);
+// }
 
-    return dev->sysfs_config_fd;
-}
+// bool reset_by_sysfs(PCIDevice *dev) {
 
-int get_revision_id(TTDevice *dev) {
+//     const char *virtual_env = getenv("VIRTUAL_ENV");
+//     if (virtual_env == nullptr)
+//         return false;
 
-    static const char pattern[] = "/sys/bus/pci/devices/%04x:%02x:%02x.%u/revision";
-    char buf[sizeof(pattern)];
-    std::snprintf(buf, sizeof(buf), pattern,
-    (unsigned int)dev->pci_domain, (unsigned int)dev->pci_bus, (unsigned int)dev->pci_device, (unsigned int)dev->pci_function);
+//     std::string reset_helper_path = virtual_env;
+//     reset_helper_path += "/bin/reset-helper";
 
-    std::ifstream revision_file(buf);
-    std::string revision_string;
-    if (std::getline(revision_file, revision_string)) {
-        return std::stoi(revision_string, nullptr, 0);
-    } else {
-        throw std::runtime_error("Revision ID read failed for device");
-    }
-}
+//     std::string busid = std::to_string(dev->pci_bus);
 
-int get_numa_node(TTDevice *dev) {
+//     dev->suspend_before_device_reset();
 
-    static const char pattern[] = "/sys/bus/pci/devices/%04x:%02x:%02x.%u/numa_node";
-    char buf[sizeof(pattern)];
-    std::snprintf(buf, sizeof(buf), pattern,
-    (unsigned int)dev->pci_domain, (unsigned int)dev->pci_bus, (unsigned int)dev->pci_device, (unsigned int)dev->pci_function);
+//     char *argv[3];
```

Comment:
Suggestion, take it or leave it: for large blocks of code that we want to comment out but leave there for reference in the near future, suggest:

```
// leaving this in for now, TODO clean up
#if 0
<code>
#endif   
```

Reply:
Changed.